base_app/post-build-step.bash

```diff
@@ -44,15 +48,15 @@ if (($# > 0)) && [[ "$1" == "--include-mocks" ]]; then
 fi
 
 # Fix names of javascript files
-cat dist/simpleui/index.html | \
+cat dist/${PROJ}/index.html | \
   awk -v JS_FOLDER="/${PROJ}/js/" -v CSS_FOLDER="/${PROJ}/css/" \
         '{ s=$0; \
          cssPrefix="<link rel=\"stylesheet\" href=\""; \
          jsPrefix="<script src=\""; \
          gsub(cssPrefix, cssPrefix CSS_FOLDER, s); \
          gsub(jsPrefix, jsPrefix JS_FOLDER, s); \
          gsub("</script>", "</script>\n", s); \
-         print s;}' > dist/index.html
+         print s;}' > dist/index.html 
```

Comment:
trailing space at the end of the line, you should set your IDE to automatically trim trailing spaces (and add a single newline to the end of the file)

Reply:
Fixed.